examples/src/examples/AfiInteractiveOptimizerEngineExample.java

```diff
@@ -30,10 +30,6 @@
 
 public class AfiInteractiveOptimizerEngineExample {
   private static FdsApiClient apiClient = null;
-  private static String BASE_PATH = "https://api.factset.com";
-  private static String USERNAME = "<username-serial>";
-  private static String PASSWORD = "<apiKey>";
```

Comment:
Let's have these variables and assign the values here by reading from the environment. That way it would be clear to the client that he needs to set some values in his environment.

Reply:
Added the environmental variable lines for all the products.